src/Microsoft.VisualStudio.Extensibility.Testing.Xunit.Shared/OutOfProcess/TestInvoker_OutOfProc.cs

```diff
@@ -84,6 +84,8 @@ public void WriteLine(string format, params object[] args)
             {
                 _testOutputHelper.WriteLine(format, args);
             }
+
+            public override object InitializeLifetimeService() => null;
```

Comment:
❔ Why is this one long-lived? It should only be valid for the duration of a single call, so won't the infrastructure keep it alive for the necessary duration?

Reply:
Hm... We have the error "IPC channel failure" (VS2019). I assumed that this is due to the fact that the object is destroyed earlier than it should. Now I can definitely confirm that this is definitely not the case for this reason. In 2019 studio, during testing, communication with the VB/CS compiler constantly disappears, which leads to a drop in the IPC channel.

Reply:
Nevertheless, we managed to get around this error on our side, so you can skip the part with the InitializeLifetimeService.

Reply:
> communication with the VB/CS compiler constantly disappears

How is this observed?

Reply:
@AndrewAdov I submitted #91 with several changes related to this